src/components/variant/ui/drawer-window.tsx

```diff
@@ -3,6 +3,7 @@ import {
   Dispatch,
```

Comment:
Seems that this changes got here by accident

Reply:
fixed